sips/pending/_posts/2017-01-11-refer-other-arguments-in-args.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: sip
+disqus: true
+title: SIP-NN - Allow referring to other arguments in default parameters
```

Comment:
I would actually write "Allow references to left arguments of the same list in default parameters" if @pathikrit agrees with my previous analysis that doing it for all the arguments is unfeasible. :smile: